easybuild/easyconfigs/h/h5py/h5py-2.9.0-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'h5py'
+version = '2.9.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.h5py.org/'
+description = """HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,
+ version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous
+ amounts of data."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'usempi': True}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['9d41ca62daf36d6b6515ab8765e4c8c4388ee18e2a665701fef2b41563821002']
+
+# to really use mpi enabled hdf5 we now seem to need a configure step
+prebuildopts = ' python setup.py configure --mpi --hdf5=$EBROOTHDF5 && '
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('SciPy-bundle', '2019.03'),
+    ('pkgconfig', '1.5.1'),
```

Comment:
build dep? (same for the all of them)

Reply:
pkgconfig is a runtime dep in this  case, and so is Python.

Reply:
I was referring to just pkgconfig; A test with an empty venv, pip only installed numpy. no pkgconfig. Also, the h5py module i installed (with pkgconfig builddep) works fine in my tests. I think the older configs were just wrong (at least, not valid for version 2.9.0)

Reply:
Ah, sorry, was thinking of pkgconfig itself... Tired